src/pynwb/image.py

```diff
@@ -94,13 +94,10 @@ def __init__(self, **kwargs):
             setattr(self, key, val)
 
         if self._change_external_file_format():
-            warnings.warn(
-                "%s '%s': The value for 'format' has been changed to 'external'. "
-                "Setting a default value for 'format' is deprecated and will be changed "
-                "to raising a ValueError in the next major release."
-                % (self.__class__.__name__, self.name),
-                DeprecationWarning,
-            )
+            self._error_on_new_warn_on_construct(error_msg=f"{self.__class__.__name__} '{self.name}': "
+                                                "The value for 'format' has been changed to 'external'. "
+                                                "If an external file is detected, setting a value for "
+                                                "'format' other than 'external' is deprecated.")
```

Comment:
I found the meaning of this `ImageSeries.format` warning a little unclear and so I was not sure how to describe here and in the CHANGELOG.  My interpretation was if an external file is added to image series, the value of format is required to be 'external'.

Reply:
that is correct